pkg/vtpm/swtpm-vtpm/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/lf-edge/eve/pkg/vtpm
 go 1.22
 
 require (
+	github.com/google/uuid v1.6.0
```

Comment:
Why not using the uuid package we use elsewhere in EVE? Makes for less dependencies with potential CVEs down the road.

Reply:
It was used only in the tests not the in vtpm server itself, but changed it to `github.com/satori/go.uuid`.